build-logic/src/main/kotlin/api.kt

```diff
@@ -75,6 +77,22 @@ fun Project.apolloLibrary(
       attributes(mapOf("Automatic-Module-Name" to namespace))
     }
   }
+
+  plugins.apply("app.cash.licensee")
+  extensions.getByType(LicenseeExtension::class.java).apply {
+    unusedAction(UnusedAction.IGNORE)
+
+    allow("Apache-2.0")
+    allow("MIT")
+    allow("CC0-1.0")
+    allow("MIT-0")
+
+    allowUrl("https://raw.githubusercontent.com/apollographql/apollo-kotlin-execution/main/LICENSE")
+    allowUrl("https://raw.githubusercontent.com/apollographql/apollo-kotlin-mockserver/main/LICENSE")
+    allowUrl("https://raw.githubusercontent.com/apollographql/apollo-kotlin/main/LICENSE")
+    allowUrl("https://asm.ow2.io/license.html")
+    allowUrl("https://spdx.org/licenses/MIT.txt")
```

Comment:
👀 surprised that this is needed when `MIT` is allowed (not a problem at all, just wondering)